src/Data/Nat/Properties.agda

```diff
@@ -1473,6 +1498,16 @@ m∸[m∸n]≡n {suc m} {suc n} (s≤s n≤m) = begin-equality
 *-distrib-∸ : _*_ DistributesOver _∸_
 *-distrib-∸ = *-distribˡ-∸ , *-distribʳ-∸
 
+2m≢1+2n :  ∀ m n → 2 * m ≢ suc (2 * n)
```

Comment:
I think a more descriptive name for this proof might be `even≢odd`

Reply:
Done.